packages/gatsby-image/src/withIEPolyfill/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, createRef } from "react"
+import PropTypes from "prop-types"
+import Image from "../index"
+
+class ImageWithIEPolyfill extends Component {
+  constructor(props) {
```

Comment:
you could go for `imageRef = createRef()` and drop the constructor, babel transpiles this correctly.